mlflow/utils/class_utils.py

```diff
@@ -1,6 +1,19 @@
+import dis
 import importlib
+import inspect
 
 
 def _get_class_from_string(fully_qualified_class_name):
     module, class_name = fully_qualified_class_name.rsplit(".", maxsplit=1)
     return getattr(importlib.import_module(module), class_name)
+
+
+def nameof(_) -> str:
```

Comment:
This is exceptionally clever. 

However, do we really want to be disassembling bytecode to get the name of a variable? 

for the current use case that this is being used for, is there a compelling downside to specifying the name of that variable as a constant?

p.s., I did grin when I saw this.

Reply:
I'm not a fan of a trick like this. Can we just hardcode?

Reply:
I completely see your point. Let's hardcode `batch_name` and revisit it when this is a frequent enough problem.